substrate-archive-common/src/util.rs

```diff
@@ -138,3 +104,37 @@ fn format_opt(file: Option<String>) -> String {
         Some(f) => f,
     }
 }
+
+/// get the path to a local substrate directory where we can save data
```

Comment:
```suggestion
/// Get the path to a local substrate directory where we can save data.
```

I think we need more info here. What is "local substrate directory" (for the various platforms)? What if I configured my node to store data elsewhere? If archive makes hard assumptions on the paths we should document that. And also mention that archive's data is in `some/path/substrate_archive`.

Reply:
It's this method: https://docs.rs/directories/3.0.1/directories/struct.BaseDirs.html#method.data_local_dir

so 

Platform | Value | Example
-- | -- | --
Linux | $XDG_DATA_HOME or $HOME/.local/share | /home/alice/.local/share
macOS | $HOME/Library/Application Support | /Users/Alice/Library/Application Support
Windows | {FOLDERID_LocalAppData} | C:\Users\Alice\AppData\Local


and I think it's only used for the log directory so far, however it is a hard-coded path so we should probably update the comment with that